src/poster.js

```diff
@@ -1,25 +1,51 @@
-import { useState } from 'react';
+import { useState, useEffect } from 'react';
+import PosterBack from './PosterBack';
 import './poster.css';
 
-function Poster({ title, image }) {
+function Poster({ title, image, id }) {
+    const [movieDetails, setMovieDetails] = useState({})
     const [flipped, setFlipped] = useState(false)
+    const [error, setError] = useState(null);
 
     const handleFlip = () => {
         setFlipped(!flipped);
+        getDetails()
       };
     
-      return (
-        <div className='Poster' onClick={handleFlip}>
+    function getDetails() {
+        fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/movies/${id}`)
+        .then((response) => {
+          if (response.ok) {
+              return response.json();
+            } else {
+              throw new Error('Server Error!');
+            }
+          })
+          .then((details) => {
+            console.log(details.movie);
```

Comment:
Be sure to remove console.logs before your final push (ideally they would never be merged into main)